ext/data/language/japanese-transforms.json

```diff
@@ -3,107 +3,107 @@
     "conditions": {
         "v": {
             "name": "Verb",
-            "partsOfSpeech": ["v1", "v5", "vk", "vs", "vz"],
             "i18n": [
                 {
                     "language": "ja",
                     "name": "動詞"
                 }
             ],
+            "isDictionaryForm": false,
```

Comment:
This feels a little odd to me, as "v" is a dictionary form. I guess it's just that we don't want people to set words as just "v" and want them to set it as one of the sub conditions? I feel like there must be a better way to name this